meshtastic/interfaces/ble/lifecycle_service.py

```
@@ -218,18 +216,26 @@ def _tracked_func() -> object:
+                # Preferred signature: safe_execute(func, error_msg=...)
                 return safe_execute(_tracked_func, error_msg=error_msg)
             except TypeError as exc:
+                # Compatibility fallbacks:
+                #   1) positional error_msg
+                #   2) callable-only legacy signature
                 if _is_unexpected_keyword_error(exc, "error_msg"):
                     try:
                         return safe_execute(_tracked_func, error_msg)
-                    except Exception:  # noqa: BLE001 - hook failures must not abort shutdown
+                    except (
+                        Exception
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)